src/components/layout/header/platform-switcher/index.tsx

```diff
@@ -1,13 +1,75 @@
+import React, { useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { DropdownMenu, DropdownMenuTrigger, DropdownMenuContent } from '../../../ui/dropdown-menu/index';
+import PlatformSwitcherCard from './platform-switcher-card';
+
+export interface TAvailableApp {
+    src: string;
+    title: string;
+    alt: string;
+    path: string;
+}
 
 const PlatformSwitcher = () => {
     const { t } = useTranslation();
+
+    const [isDropdownOpen, setDropdownOpen] = useState<boolean>(false);
```

Comment:
```suggestion
    const [isDropdownOpen, setDropdownOpen] = useState(false);
```

Don't need type here, cause it's initial value is boolean so it automatically accept that type!